src/sentry/static/sentry/app/views/settings/project/projectOwnership/ruleBuilder.jsx

```diff
@@ -74,7 +80,11 @@ class RuleBuilder extends React.Component {
       )
       .join(' ');
 
-    const rule = `${type}:${text} ${ownerText}`;
+    const quotedText = text.match(/\s/) ? `"${text}"` : text;
+
+    const rule = `${
+      type === 'tag' ? `tags.${tagName}` : type
```

Comment:
shouldn't type be an ENUM?

Reply:
Dunno. Just extended what was already there.